src/app/problem/editor/right/testcases/usecase.js

```diff
@@ -4,40 +4,51 @@ import {TestcaseIOsPatch} from "../../../../../services/ProblemService";
 
 let currentTestcaseEdits = [];
 
-const useTestcaseEditList = (problem) => {
-    const [testcaseEdits, setTestcaseEdits] = useState([]);
+const useTestcaseEditList = () => {
+    const [problem,  setProblem] = useState(undefined);
+    const [testcaseEdits, setTestcaseEdits] = useState(undefined);
 
     const addNewTestcase = () => {
-        const nextId = testcaseEdits.length === 0 ? 0 : (Math.max(...testcaseEdits.map(t => t.id)) + 1);
-        setTestcaseEdits([...testcaseEdits,
-            new TestcaseEdit({
-                id: nextId.toString(),
-                name: nextId.toString(),
-                timeLimit: 1,
-                memoryLimit: 1,
-                outputLimit: 1,
-                threadNumberLimit: -1,  // currently doesn't support threadNumberLimit's modification
-                grade: 0,
-                problemId: problem.id,
-
-                editing: true // new testcase --> editing it immediately
-            })]);
+        const nextId = testcaseEdits.length === 0 ? 0 : generateNextTestcaseId(testcaseEdits);
+        const newTestcaseEdit = new TestcaseEdit({
+            id: nextId.toString(),
+            name: nextId.toString(),
+            timeLimit: 1,
+            memoryLimit: 1,
+            outputLimit: 1,
```

Comment:
In the current Judge Girl, it has `timeLimit: 1.0s` and `memoryLimit: 64MB` by default.
Please update these two limitations to the correct default value.